config/routes.rb

```diff
@@ -21,5 +21,10 @@
   get "/422", to: "errors#unprocessable_entity", via: :all
   get "/500", to: "errors#internal_server_error", via: :all
 
+  if Rails.env.development?
```

Comment:
Do we also want this on deployed_development?

Reply:
Good shout!